lib/v2/yzw/kydt.js

```diff
@@ -0,0 +1,54 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+const host = 'https://yz.chsi.com.cn';
+
+module.exports = async (ctx) => {
+    const response = await got({
+        method: 'get',
+        url: `${host}/kyzx/kydt`,
+    });
```

Comment:
Please keep using `got` in 1 style only.
Either
```js
await got({
    // ...options
})
```
or like
https://github.com/DIYgod/RSSHub/blob/30e3bb61d98d3aa5a908872f0e481475cc0b0c3a/lib/v2/yzw/kydt.js#L32